query-service-impl/src/main/java/org/hypertrace/core/query/service/postgres/PostgresBasedRequestHandler.java

```diff
@@ -196,6 +205,13 @@ public QueryCost canHandle(QueryRequest request, ExecutionContext executionConte
       cost = tableDefinition.getTimeGranularityMillis() / (Long.MAX_VALUE * 2D);
     }
 
+    long requestEndTime = getRequestEndTime(request.getFilter());
+    Duration requestDuration = Duration.ofMillis(requestEndTime - requestStartTime);
+
+    if (minRequestDuration != Duration.ZERO
```

Comment:
Why do we need to check for Duration Zero explicitly. Single compare check as below should be good enough. We will anyways have order defined in query service config, and handler will higher minDuration will be defined first.

Reply:
We can do that. I was keeping the default handler first in the list. 

With this, we have to keep the handler order from highest to lowest minRequestDuration. Logically, this is easy to maintain, addressing the comments.

e.g if there are three handlers as described in test cases, we will have to list them in the following order.
- backend-traces-from-bare-span-event-view-5min-aggr-handler
    - minRequestDuration : 12 hrs
- backend-traces-from-bare-span-event-view-30secs-aggr-handler
    - minRequestDuration : 3 hrs
- backend-traces-from-bare-span-event-view-aggr-handler 
    - minRequestDuration : 0 hrs

Reply:
Done.